### PR TITLE
Milestone 1: Building with Responsive Design is done

### PR DESCRIPTION
**Pull request by Alexoid1**
## This project is a clone of the (https://thenextweb.com/) website , to recreate all the features and resposivity of the website.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,841 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet"
+        integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Nova+Square&display=swap" rel="stylesheet">
+    <!-- <link rel="stylesheet" href="styles/reset.css"> -->
```
Comment (BrittanyBlake) [inline index.html]: please remove commented code.

File: styles/styleForLargeScreen.css [hunk 2]
```diff
@@ -0,0 +1,837 @@
...
+
+.rightCoverArticle {
+  background: linear-gradient(180deg, rgba(27, 112, 186, 1) 0%, rgba(68, 152, 217, 0.11528361344537819) 100%), url(https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/05/Tony-Hawk-796x417.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: flex-between;
```
Comment (BrittanyBlake) [inline styles/styleForLargeScreen.css]: `flex-between` is not a valid `jusify-content` value

File: styles/styleForLargeScreen.css [hunk 3]
```diff
@@ -0,0 +1,837 @@
...
+.articleNameAndIcons2 {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: flex-start;
+  padding-left: 10px;
+}
+
+.articleNameAndIcons2 h3 {
+  size: 10px;
```
Comment (BrittanyBlake) [inline styles/styleForLargeScreen.css:326]: the property `size` does not exist for this. Did you mean font-size?

File: styles/styleForMediumScreen.css [hunk 4]
```diff
@@ -0,0 +1,844 @@
...
+
+.rightCoverArticle {
+  background: linear-gradient(180deg, rgba(27, 112, 186, 1) 0%, rgba(68, 152, 217, 0.11528361344537819) 100%), url(https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/05/Tony-Hawk-796x417.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: flex-between;
```
Comment (BrittanyBlake) [inline styles/styleForMediumScreen.css]: flex-between is not a justify-content value

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
## Status: Changes Required ♻️ 
****
### Hi @Alexoid1 and @RaminMammadzada! 👋  👋 
Your project looks amazing and your repository looks professional 👏 👏    There are just a few issues that need to be resolved before you can move on: ⬇️ 
****
### General

- [ ] Please remove all instances of commented code. It is bad practice to leave commented code in your files as it makes your project look untidy. If you need to view an older version of your code you can use your commit history. 
Screen Shot 2020-06-27 at 2 38 32 AM
- [ ] Please see my comments above for additional changes to be made. 



### Navigation Bar
- **Medium version**

- [ ] Great job making the TNW logo responsive! 👍 However, the logo should also span the height of the top (black) navigation bar. Please compare your clone with the original page and fix this.
Your clone 👇 
Screen Shot 2020-06-27 at 2 09 58 AM
The original page 👇 
Screen Shot 2020-06-27 at 2 10 21 AM

- **Small version**
- [ ] A hamburger icon should appear at the rightmost part of the navigation bar, next to the search icon. Additionally, the TNW logo should also remain on the small screen size. Please compare your clone with the original and fix this. 
Your clone 👇 
Screen Shot 2020-06-27 at 2 12 44 AM
The original page 👇 
Screen Shot 2020-06-27 at 2 13 05 AM




### Top 3 articles section

- **Medium version**

- [x] In the medium screen, the one bigger image on the left should continue to take up around two-thirds of the width, just as it does on the large screen. Please compare your clone and the original and fix this. 
Your clone 👇 
Screen Shot 2020-06-27 at 2 16 39 AM
The original page 👇 
Screen Shot 2020-06-27 at 2 17 15 AM


### Latest news section
- **Small version**

- [x] There is too much empty space between the "Latest News" heading and the articles below. 
Your clone 👇 
Screen Shot 2020-06-27 at 2 20 04 AM
The original page 👇 
Screen Shot 2020-06-27 at 2 20 44 AM

- [x] There is also too much space between the article images and text. 
The original page 👇 
Screen Shot 2020-06-27 at 2 21 28 AM
Your clone 👇 
Screen Shot 2020-06-27 at 2 21 45 AM



### Latest funding section
- **Medium version**

- [x] The links (Add info, get updates, see more) should each be on their own line in the medium screen. Please compare your clone with the original in the screenshots below and fix this. 
Your clone 👇 
Screen Shot 2020-06-27 at 2 24 04 AM
The original page 👇 
Screen Shot 2020-06-27 at 2 24 29 AM

### Categories (Apps, Gear, Tech, etc) section

- **Large version** 

- [x] The last two articles should be left aligned, while on your clone they appear to be centered. Please fix this.
Your clone 👇 
Screen Shot 2020-06-27 at 2 28 39 AM
The original page 👇 
Screen Shot 2020-06-27 at 2 29 06 AM


- **Medium version**

- [x] In the medium screen, it should align to two category boxes per row while yours currently aligns to one. Please compare your clone with the original and fix this. 
Your clone 👇 
Screen Shot 2020-06-27 at 2 26 32 AM
The original page 👇 
Screen Shot 2020-06-27 at 2 26 52 AM
### Latest deals section

- **Medium version**

- [x] The header and the "see all deals" link are not properly aligned. 
Your clone 👇 
Screen Shot 2020-06-27 at 2 30 39 AM
The original page 👇 
Screen Shot 2020-06-27 at 2 31 07 AM

- **Small version**
- [x] The header and the "see all deals" link are not properly aligned. Additionally, there is too much space between the header and the deals.
Your clone 👇 
Screen Shot 2020-06-27 at 2 33 23 AM

The original page 👇 
Screen Shot 2020-06-27 at 2 33 12 AM

- [x] The deal images are not properly formatted on this screen size. They are rectangular and have a much larger height than the original.
The original page 👇 
Screen Shot 2020-06-27 at 2 34 28 AM
Your clone 👇
Screen Shot 2020-06-27 at 2 34 57 AM
 
****

### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by RaminMammadzada** [comment]
> ### Latest funding section
> * **Medium version**
> * [x]  The links (Add info, get updates, see more) should each be on their own line in the medium screen. Please compare your clone with the original in the screenshots below and fix this.
>   Your clone 👇
>   Screen Shot 2020-06-27 at 2 24 04 AM
>   The original page 👇
>   Screen Shot 2020-06-27 at 2 24 29 AM
> 
> _P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

In the "Latest Funding Round" part, for the Medium size of the original website the "sublist" is set horizontally below the header, not vertically as you send in the screenshot. I also got the screenshot from the original, you can check it below:

image

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🛑

## Hello team! Thanks for making the changes. But you have missed some as well.

- The logo in tablet mode is still the same.

#### Clone
image

#### Original
image

In addition to above:

- The clone is not full width in mobile view.

#### Clone
Annotation 2020-06-27 170757

#### Original
image

Apply these changes and request another review.

Happy coding 👨‍💻

**Review by elmejdki — APPROVED** [review]
# Great job @team :clap: you have made a good clone, thank you for your patients and thanks also for making the required changes by other TSE's

# status: approved

now you can merge to master, your project is approved

### Happy Coding.

**Pull request merged**